.gitignore

```diff
@@ -0,0 +1,4 @@
+_site
+CNAME
```

Comment:
@denisshevchenko I don't know. I just copied `.gitignore` from previous project. Maybe there were reasons. If there no reasons I can remove this line.